gensim/models/doc2vec.py

```diff
@@ -21,17 +21,21 @@
 >>> model.save(fname)
 >>> model = Doc2Vec.load(fname)  # you can continue training with the loaded model!
 
-The word vectors in the model can also be instantiated from an existing file on disk in the word2vec C format:
-NOTE: document vectors are not loaded/saved with .load/save_word2vec_format(). Use .save()/.load() instead.
+If you're finished training a model (=no more updates, only querying), you can do
+
+  >>> model.delete_temporary_training_data(keep_doctags_vectors=True, keep_inference=True):
```

Comment:
I see, though this is still kind of odd. As called in this prominent example, this method hardly gets rid of anything – just the relatively-tiny `doctag_syn0_lockf`. Someone who just needs that tiny benefit could be coached to execute `del model.docvecs.doctag_syn0_lockf`. (I fear here, and to some extend on Word2Vec too, this method is attractive to novices but likely to cause headaches for them and then support/maintenance issues down the road.)